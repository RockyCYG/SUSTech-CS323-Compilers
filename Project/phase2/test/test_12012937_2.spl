char k(){}
int aa(float x)
{
	return x;
}
struct A{
};
struct B{
	struct A aduf;
};
struct C{

	struct A auuu;
	struct B bwww;
	struct C c;
};
struct C{};
int C(){}
int C(int a,int b){}
int gay()
{
	struct C c22[10];
	struct C deeee;
	c22[0].c.c.c=(deeee.c.c*deeee.c.c).c.c.c;
}
int test_array()
{
	float a55;
	float b55;
	float c55;
	aa(c55=(a55=b55)*(a55==b55));
	aa(a55=b55)=(a55==b55);
}
struct B strc(struct A a1,struct A b1){
	struct B c1;
	c1.aduf=a1==b1;
	return c1;
}
int cpp()
{
	struct A ababa;
	struct A babab;
	ababa=babab;
}
int t0()
{
	int i;
	int j;
	char yy;
	int o;
	i=j&&o+j+o*i+i>=o;
	yy=yy+i;
}
