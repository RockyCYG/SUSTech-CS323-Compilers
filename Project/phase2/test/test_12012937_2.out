Error type 8 at Line 4: incompatiable return type
Error type 15 at Line 17: redefine struct: C
Error type 4 at Line 19: redefine function: C
Error type 6 at Line 32: left side in assignment is rvalue
Error type 7 at Line 36: binary operation on non-number variables
Error type 7 at Line 52: binary operation on non-number variables

